pysumma/simulation.py

```diff
@@ -64,22 +93,25 @@ def gen_summa_cmd(self, processes=1, prerun_cmds=[],
 
         return preprocess_cmd + summa_run_cmd
 
-    def run_local(self, run_suffix=None, processes=1, prerun_cmds=[],
-                  startGRU=None, countGRU=None, iHRU=None, freq_restart=None,
-                  progress=None):
-        run_cmd = self.gen_summa_cmd(processes, prerun_cmds,
-                                     startGRU, countGRU, iHRU, freq_restart,
-                                     progress)
+    def _run_local(self, run_suffix, processes=1, prerun_cmds=None,
+                   startGRU=None, countGRU=None, iHRU=None, freq_restart=None,
+                   progress=None):
+        """Start a local simulation"""
+        run_cmd = self._gen_summa_cmd(run_suffix, processes, prerun_cmds,
+                                      startGRU, countGRU, iHRU, freq_restart,
+                                      progress)
         self.process = subprocess.Popen(run_cmd, stdout=subprocess.PIPE,
                                         stderr=subprocess.PIPE, shell=True)
-        self._status = 'Running'
+        self.status = 'Running'
 
-    def run_docker(self, run_suffix=None, processes=1,
-                   prerun_cmds=[], startGRU=None, countGRU=None, iHRU=None,
-                   freq_restart=None, progress=None):
-        run_cmd = self.gen_summa_cmd(processes, prerun_cmds,
-                                     startGRU, countGRU, iHRU,
-                                     freq_restart, progress)
+    def _run_docker(self, run_suffix, processes=1,
```

Comment:
![image](https://user-images.githubusercontent.com/23341664/58591298-f6f97f80-8233-11e9-9d7d-8e081efc9ae2.png)

This is a docker running error.

Reply:
Could you provide more information about how you set up the simulation?